platform-configs/protectli-vp4650.robot

```diff
@@ -178,6 +182,9 @@ ${RESET_TO_DEFAULTS_SUPPORT}=                       ${TRUE}
 ${MEMORY_PROFILE_SUPPORT}=                          ${FALSE}
 ${DEFAULT_POWER_STATE_AFTER_FAIL}=                  Powered Off
 ${ESP_SCANNING_SUPPORT}=                            ${FALSE}
+${UUID_SETTABLE}=                                   ${TRUE}
+${SERIAL_NUMBER_SETTABLE}=                          ${TRUE}
+${BOOTSPLASH_SETTABLE}=                             ${TRUE}
 
```

Comment:
When introducing new flags, it must be added to all configs.

Reply:
Added in commit 719854125560f4903705bec5d7ef8641d0ee14e4